src/ripple/app/misc/Validations.cpp

```diff
@@ -387,6 +387,39 @@ class ValidationsImp : public Validations
         return ret;
     }
 
+    std::uint32_t getValidationTime (uint256 const& hash, int minValidations)
```

Comment:
Its definitely an alternative, and one that I considered before returning the `vector`. The problem is that it makes the `Validations` interface care about `Ledger`.